rclcpp/include/rclcpp/parameter_service.hpp

```diff
@@ -39,12 +40,26 @@ class ParameterService
 public:
   RCLCPP_SMART_PTR_DEFINITIONS(ParameterService)
 
+  [[deprecated("use rclcpp::QoS instead of rmw_qos_profile_t")]]
```

Comment:
I was going to mention this, i think this should also be deprecated accordingly. but it is not included in original description https://github.com/ros2/rclcpp/issues/1967#issue-1299387343. CC: @sloretz

Reply:
About this, I refer to changes in [Make create_client accept rclcpp::QoS](https://github.com/ros2/rclcpp/pull/1964/files#top).